public/_locales/en/messages.json

```diff
@@ -577,7 +577,15 @@
     "message": "Inclusion of at least one special character, e.g., ! @ # ? ] - (optional, recommended)."
   },
   "mnemonicWordsAmountConstraint": {
-    "message": "12, 15, 18, 21 or 24 words in English"
```

Comment:
Please check how the application works with other languages. I'm afraid an error will be thrown because of difference in translation arguments

Reply:
it works fine

Reply:
> Please check how the application works with other languages. I'm afraid an error will be thrown because of difference in translation arguments

Reply:
> it works fine

Reply:
> > Please check how the application works with other languages. I'm afraid an error will be thrown because of difference in translation arguments
> 
>

Reply:
> > it works fine
> 
>